nilearn/datasets/description/nki_enhanced_surface.rst

```diff
@@ -1 +1,22 @@
+NKI enhanced surface
 
+
+Notes
+-----
+NKI enhanced resting-state dataset, preprocessed and projected to the 
```

Comment:
I'll need to write a bit more about which data of eNKI this is, we should also include the link to the preprocessing pipeline